packages/fast-components-styles-msft/src/divider/index.ts

```diff
@@ -18,7 +18,7 @@ const styles: ComponentStyles<DividerClassNameContract, DesignSystem> = (
         divider: {
             boxSizing: "content-box",
             height: "0",
-            margin: `${density(designSystem.designUnit * 3)(designSystem)} 0`,
+            margin: `0`,
```

Comment:
nit: I know this works, but we don't actually need back-ticks here and would prefer to use quotes in these cases instead.